src/components/KonnectorList.jsx

```diff
@@ -5,11 +5,39 @@ import { popupCenter } from '../lib/popup'
 
 const VOTING_LINK = 'https://framaforms.org/cozy-collect-1494574386'
 
-const KonnectorList = ({ t, connectors, showVoting = false }) => (
+const getAccountName = account => {
```

Comment:
I feel that this logic should be elsewhere, maybe in the store, and be something like a selector. After the redux migration I guess that it will be something like that. What do you think ?